_posts/2021-03-29-apply-pretrained-pytorch-model.md

```diff
@@ -0,0 +1,154 @@
+---
+layout: post
+title: Dask with PyTorch for large scale image analysis
+author: Nicholas Sofroniew, Matthew Rocklin
```

Comment:
I'm not sure I did anything here.  I'm pretty sure that @GenevieveBuckley did.  Probably we should tweak the author list appropriately :)

Reply:
Ok, I can change that for you. For the record though, I think my contribution here is mostly just insisting we not let Nick's work languish.